CHARTER.md

```diff
@@ -31,13 +31,65 @@ The membership profile is…
 ### 2.2 Community Feedback
 
 
-## 3. Meetings
+## 3. Communication
 
-### 3.1 Cadence
+### 3.1 Channels
 
-### 3.2 Annual Activities 
+Asynchronous public communication is preferred for working group activities. This section attempts to identify the communication channels along with recommendations for how and when we'll use them.
 
-Ex: budget/finance, meeting planning, elections, etc…
+*Mailing List*
+- monthly newsletter summarizing working group activities
+- used by managers for any working group announcements
+- low volume
+
+*GitHub*
+- **GitHub Discussions** used for working group feedback, suggestions, and polls
+- **GitHub Issues** used to track working group tasks
+- **GitHub Projects** used to track multiple tasks into an outcome-driven kanban aligned with working group strategic goals
+- **GitHub Pull Requests** provide opportunity for feedback from participants on any working group operating document changes
+
+*Discord*
+- use the #resource-aggregation channel for working group specific chat
+- agenda and summary of any working group calls
+- higher frequency of updates to call attention to activity from other channels
+
+*Zoom*
+- see the next section for more about working group calls
+
+### 3.2 Working Group Calls
+
+Real-time communication will utilize the community calendar to announce events to be held using Zoom for teleconferencing.
```

Comment:
Is using Zoom a strong requirement in DRF?

Reply:
Good question. I have no preference, but I thought I saw that was one that DRF already has access to as a resource and has been using for community calls.

Reply:
LF hosted projects are moving to the PCC LFX to improve collaboration and communication with role-based access control: https://docs.linuxfoundation.org/lfx/project-control-center/v2-latest-version/overview

So yes, Zoom and groups io should be used, as they are the tools that LFX PCC has integrated when scheduling meetings, assigning community hosts, inviting people from the working group.io mailing lists, etc.

Reply:
Here is the doc re: meetings 🙂 https://docs.linuxfoundation.org/lfx/project-control-center/v2-latest-version/collaborations/meetings

Reply:
I put in #22 to capture this under resources available for the working group. Thanks @anajsana.